src/components/file/editForm/File.edit.file.js

```diff
@@ -170,6 +170,14 @@ export default [
     tooltip: 'This will allow using an attached camera to directly take a picture instead of uploading an existing file.',
     weight: 32
   },
+  {
```

Comment:
Shouldn't this be just the default behavior? Why are we adding a configuration for this?

Reply:
Okay, made this behavior default.